quote-generator/src/app/extensions/components/TripDetails.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import {
+  Flex,
+  Form,
+  NumberInput,
+  Button,
+  ButtonRow,
+} from '@hubspot/ui-extensions';
+
+export const TripDetails = ({ onNext }) => {
+  const [passengers, setPassengers] = useState(10);
```

Comment:
nit: similar to what you did above: having the default values in constant with descriptive names is quite nice